packages/hoprd/releases.json

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    "id": "master-goerli",
+    "git_ref": "refs/heads/master",
+    "environment_id": "master-goerli"
+  },
+  {
+    "id": "wildhorn",
+    "version_major": 1,
+    "version_minor": 76,
+    "git_ref": "refs/heads/release/wildhorn",
+    "environment_id": "wildhorn"
+  },
+  {
+    "id": "constantine",
+    "version_major": 1,
+    "version_minor": 75,
+    "git_ref": "refs/heads/release/constantine",
```

Comment:
environment_id missing, so is that part of deprecating a release or should be keep an environment around which could be used to build this release?